src/screens/contacts/ContactsList.js

```diff
@@ -0,0 +1,116 @@
+import {Image, StyleSheet, Text, View, FlatList} from 'react-native';
+import React from 'react';
+import Logo from '../../components/Logo';
+
+const allContacts = [
+  {
+    id: 1,
+    name: 'Umar',
+    email: 'umar@gmail.com',
+    DOB: '12/04/1980',
+    phone: '+92-31765908',
+  },
+  {
+    id: 2,
+    name: 'Jake',
+    email: 'jake@gmail.com',
+    DOB: '11/09/1960',
+    phone: '+98-31865456',
+  },
+  {
+    id: 3,
+    name: 'Bob',
+    email: 'bob@gmail.com',
+    DOB: '03/09/1890',
+    phone: '+92-434659654',
+  },
+  {
+    id: 4,
+    name: 'Meghan',
+    email: 'meghan@gmail.com',
+    DOB: '06/04/2010',
+    phone: '+99-3143590987',
+  },
+];
+
+const ContactItem = ({contact}) => (
+  <View style={styles.userContainer}>
+    <Image
+      style={styles.profileImage}
+      source={require('../../assets/images/dummy_image.png')}
+    />
+    <View style={styles.userDataContainer}>
+      <Text style={styles.userName}>{contact.name}</Text>
+      <Text style={styles.userEmail}>{contact.email}</Text>
+      <Text style={styles.phone}>{contact.phone}</Text>
+    </View>
+    <Image
+      style={styles.right_arrow}
+      source={require('../../assets/images/right_arrow.png')}
+    />
+  </View>
+);
+
+const ContactsList = () => {
+  const renderContact = ({item}) => <ContactItem contact={item} />;
+  return (
+    <>
+      <View style={{paddingHorizontal: 33}}>
+        <Logo />
+        <FlatList
+          data={allContacts}
+          renderItem={renderContact}
+          keyExtractor={item => item.id}
+        />
+      </View>
+    </>
+  );
+};
```

Comment:
Keep your main function at top and child at bottom, it will help you in readability

Reply:
mark it as resolved